api/tests/database/migrations/run.test.ts

```diff
@@ -16,7 +16,7 @@ describe('run', () => {
 	});
 
 	describe('when passed the argument up', () => {
-		it('returns "Nothing To Updage" if no directus_migrations', async () => {
+		it('returns "Nothing To Update" if no directus_migrations', async () => {
```

Comment:
```suggestion
		it('returns "Nothing To Upgrade" if no directus_migrations', async () => {
```

Ref line 23 in this test file and where the error is thrown:

https://github.com/directus/directus/blob/70570a5f75188bee75f7895be489fa272acdce42/api/src/database/migrations/run.ts#L62-L64